index.yaml

```diff
@@ -2,30 +2,6 @@
 # ROS buildfarm index file
 ---
 distributions:
-#  r2b2:
-#    doc_builds: {}
-#    notification_emails:
-#      - stevenragnarok@osrfoundation.org
-#    release_builds:
-#      default: r2b2/release-build.yaml
-#      uxv8: r2b2/release-xenial-arm64-build.yaml
-#    source_builds: {}
-#  r2b3:
-#    doc_builds: {}
-#    notification_emails:
-#      - stevenragnarok@osrfoundation.org
-#    release_builds:
-#      default: r2b3/release-build.yaml
-#      uxv8: r2b3/release-xenial-arm64-build.yaml
-#    source_builds: {}
-#  ardent:
-#    doc_builds: {}
-#    notification_emails:
-#      - steven+build.ros2.org@openrobotics.org
-#    release_builds:
-#      default: ardent/release-build.yaml
-#      uxv8: ardent/release-xenial-arm64-build.yaml
-#    source_builds: {}
   bouncy:
     doc_builds: {}
     notification_emails:
```

Comment:
Unrelated to this patch but it would be good to fix the Bouncy notification email address.

Reply:
Done as part of https://github.com/ros2/ros_buildfarm_config/pull/18